iOS/DuckDuckGo/AIChat/InputBox/SwitchBar/OmniBarEditingStateViewController.swift

```diff
@@ -109,6 +109,7 @@ final class OmniBarEditingStateViewController: UIViewController, OmniBarEditingS
         super.viewDidAppear(animated)
 
         DailyPixel.fireDailyAndCount(pixel: .aiChatInternalSwitchBarDisplayed)
```

Comment:
I know here are we already firing a similar pixel. But, I would like to go forward with consistent pixel names, and also don’t think `internal` should be in the name. So, bit of temporary duplication in pixels here is ok IMO.